fetch.bs

```diff
@@ -5993,7 +5993,7 @@ new Headers(meta2);
   <var>object</var>:
 
   <ol>
-   <li><p>If <var>header</var> does not contain exactly two items, then <a>throw</a> a
+   <li><p>If <var>header</var> is not a <a>sequence</a> which contains exactly two items, then <a>throw</a> a
```

Comment:
I looked at this again to be sure and it doesn't seem to be needed. IDL wouldn't allow this if it were not a sequence. I would be okay with adding a step before this one that asserts it's a sequence though.